cmd2/cmd2.py

```diff
@@ -1648,6 +1574,11 @@ def complete(self, text, state):
 
                 # Parse the command line
                 command, args, expanded_line = self.parseline(line)
+                
```

Comment:
Recommending setting IDE/editor to automatically strip trailing whitespace on save

Reply:
Fixed and pushed to `ply` branch.